cli/src/commit_templater.rs

```diff
@@ -2446,6 +2462,15 @@ fn builtin_tree_entry_methods<'repo>() -> CommitTemplateBuildMethodFnMap<'repo,
             Ok(out_property.into_dyn_wrapped())
         },
     );
+    map.insert(
+        "num_conflict_sides",
```

Comment:
nit: I slightly prefer `conflict_sides` or `conflict_side_count`, but I don't feel strongly.

Another idea is to add `added`/`removed_entries() -> List<TreeEntry>`.